### ISSUE TITLE
v1.6.6: ImportError: cannot import name extract_node

### ISSUE THREAD

**Issue by dagwieers** [PyCQA/astroid#736]
There is a problem introduced in v1.6.6 causing:
```
= Starting sanity pylint test
python -m pylint resources/lib/ test/
Traceback (most recent call last):
  File "/usr/lib64/python2.7/runpy.py", line 162, in _run_module_as_main
    "__main__", fname, loader, pkg_name)
  File "/usr/lib64/python2.7/runpy.py", line 72, in _run_code
    exec code in run_globals
  File "/usr/lib/python2.7/site-packages/pylint/__main__.py", line 7, in <module>
    pylint.run_pylint()
  File "/usr/lib/python2.7/site-packages/pylint/__init__.py", line 15, in run_pylint
    from pylint.lint import Run
  File "/usr/lib/python2.7/site-packages/pylint/lint.py", line 64, in <module>
    import astroid
  File "/usr/lib/python2.7/site-packages/astroid/__init__.py", line 136, in <module>
    __import__(module[:-3])
  File "/usr/lib/python2.7/site-packages/astroid/brain/brain_functools.py", line 7, in <module>
    from astroid import extract_node
ImportError: cannot import name extract_node
```

This is because `extract_node` is imported from `astroid` instead of `astroid.builder`. It seems v1.6.5 was not affected by this.
https://github.com/PyCQA/astroid/blob/af8fe7a4657be65e8087e9e4a68e6390d8ea38f3/astroid/brain/brain_functools.py#L7

**Comment by dagwieers** [comment]
Forget about this report, somehow my installation was broken. Reverting to v1.6.5 fixed the issue, going back to v1.6.6 seems to be working as well. I have no clue how I got into this broken state using pip?

**Comment by dagwieers** [comment]
Aha, might be an RPM package update messing with my pip installation :-/
```
[dag@moria service.upnext]$ rpm -V python2-astroid-1.4.9-4.el7.noarch
S.5....T.    /usr/lib/python2.7/site-packages/astroid/__init__.py
S.5....T.    /usr/lib/python2.7/site-packages/astroid/__init__.pyc
missing     /usr/lib/python2.7/site-packages/astroid/__init__.pyo
...
```

And:
```
Install Date: di 17 dec 2019 16:21:41 CET
```
